airbyte-integrations/connectors/source-twilio/source_twilio/streams.py

```diff
@@ -333,6 +351,7 @@ class Messages(TwilioNestedStream, IncrementalTwilioStream):
     parent_stream = Accounts
     incremental_filter_field = "DateSent>"
     cursor_field = "date_sent"
+    page_size = config
```

Comment:
I don't get why you set the `page_size` class attribute here... `config` is a `disutil` imported object, I think this a typo.